pkg/datapath/linux/probes/probes.go

```diff
@@ -240,6 +240,49 @@ func (p *ProbeManager) SystemConfigProbes() error {
 	return nil
 }
 
+// ExitKernelCheck exits execution of cilium CLI kernel-check early in case the config file
+// is missing (or running on a kernel before 3.18)
+func (p *ProbeManager) ExitKernelCheck() error {
+	config := p.features.SystemConfig
+
+	if config.ConfigBpf == "" {
+		return errors.New("Unable to detect BPF support. Is the kernel config available?")
+	}
+	return nil
+}
+
+// GetRequiredConfig performs a check of mandatory kernel configuration options. It
+// returns a map indicating which required kernel paramaters are enabled - and which are not
+// GetRequiredConfig is being used by CLI "cilium kernel-check"
+func (p *ProbeManager) GetRequiredConfig() (supported map[KernelParam]bool) {
```

Comment:
```suggestion
func (p *ProbeManager) GetRequiredConfig() map[KernelParam]bool {
```

You are not using the name you assign to the return value, let's drop it.